payments-core/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -60,6 +61,10 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
     protected val _paymentMethods = MutableLiveData<List<PaymentMethod>>()
     internal val paymentMethods: LiveData<List<PaymentMethod>> = _paymentMethods
 
+    @VisibleForTesting
+    internal val _amount = MutableLiveData<Amount>()
+    internal val amount: LiveData<Amount> = _amount
+
     /**
      * Request to retrieve the value from the repository happens when initialize any fragment
      * and any fragment will re-update when the result comes back.
```

Comment:
Are you finding that without a shipping address in the PaymentIntent AfterPay still works?  Originally we were thinking that it would not be shown if there was not shipping in the PaymentIntent.